ve_details.tex

```diff
@@ -0,0 +1,2698 @@
+} }
+& {\small Description} &
+\begin{minipage}[t]{13cm}{\scriptsize
+~Analyze the network logs and monitoring system to determine average and
+peak latency and packet loss statistics.
+
+\vspace{\dp0}
+} \end{minipage} \\ \cdashline{2-3}
+& {\small Test Data} &
+\\ \cdashline{2-3}
+& {\small Expected Result} &
+\begin{minipage}[t]{13cm}{\scriptsize
+Data can be transferred within the required time, i.e. verify that link
+is capable of transferring image for prompt processing in
+oArchiveMaxTransferTime = 5{[}second{]}. Verify transfer of data at or
+exceeding rates specified in LDM-142 at least 98\% of the time.
+
+\vspace{\dp0}
+} \end{minipage}
+\\ \hdashline
+        \\ \midrule
+    \end{longtable}
+\subsection{[LVV-82] DMS-REQ-0181-V-01: Base to Archive Network Availability }\label{lvv-82}
+
+\begin{longtable}{ccccc}
+\hline
+\textbf{Jira Link} & \textbf{Assignee} & \textbf{Status} & \textbf{Priority} & \textbf{Test Cases}\\ \hline
+\href{https://jira.lsstcorp.org/browse/LVV-82}{LVV-82} &
+Robert Gruendl & Not Covered & 1a &
+\begin{tabular}{c}
+LVV-T194 \\
+\end{tabular}
+\\
+\hline
+\end{longtable}
+
+\textbf{Verification Element Description:} \\
+This requirement is verified by transferring data between base and
+archive over uninterrupted 1 week period, modeling to extrapolate to an
+annual failure rate, and verifying that is within the requirement.
+
+{\footnotesize
+\begin{longtable}{p{3cm}p{13cm}}
+\hline
+\multicolumn{2}{c}{\textbf{Upstream Requirements}}\\ \hline
+Requirement ID & DMS-REQ-0181 \\ \cdashline{1-2}
+Requirement Description & \textbf{Specification:} The Base to Archive communications shall be
+highly available, with MTBF \textgreater{} \textbf{baseToArchNetMTBF}. \\ \cdashline{1-2}
+Requirement Parameters & \textbf{baseToArchNetMTBF = 180{{[}day{]}}} Mean time between failures,
+measured over a 1-yr period. \\ \cdashline{1-2}
+Requirement Priority & 1b \\ \cdashline{1-2}
+Upper Level Requirement &
+\begin{tabular}{cl}
+OSS-REQ-0053 & Base-Archive Connectivity Loss \\
+DMS-REQ-0162 & Pipeline Throughput \\
+DMS-REQ-0161 & Optimization of Cost, Reliability and Availability in Order \\
+\end{tabular}
+\\ \hline
+\end{longtable}
+}
+
+\subsubsection{[LVV-T194] Verify implementation of Base to Archive Network Availability }
+
+\begin{longtable}{cccccc}
+\hline
+\multicolumn{6}{c}{\textbf{Test Case Summary}} \\ \hline
+\textbf{Jira Link} & \textbf{Owner} & \textbf{Status} & \textbf{Version} & \textbf{Critical Event} &
+\textbf{Verification Type} \\ \hline
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T194}{LVV-T194} &
+Jeff Kantor & Draft & 1 & false & Test
+\\ \hline
+\end{longtable}
+
+\textbf{Objective:} \\
+From test and historical data, extrapolate to a full year to estimate if
+expect to meet baseToArchNetMTBF = 180{[}day{]}. ~Note that this is for
+complete loss of transfer service (simultaneous failure on all paths),
+not a single path failure with successful fail-over.
+
```

Comment:
Verify the availability of the Base to Archive Network communications by demonstrating that it  meets or exceeds a mean time between failures, measured over a 1-yr period of   MTBF > baseToArchNetMTBF (180[day])